### PR TITLE
MapLibre - Vue3 - Vite Migration v2

### PR DESCRIPTION
**Pull request by mapl**
Hi everyone, this is my 2nd attempt to migrate the internal UI Viewer project.

All the UI (dev) libraries have been updated to the most recent versions.

Mapbox -> MapLibre
Vue2 -> Vue3
Vue Cli -> Vite

### CODE REVIEW

File: ui/src/globals/store.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import Vue from "vue";
+// import Vue from "vue";
```
Comment (iwpnd) [inline ui/src/globals/store.js:1]: lets clean this up

**Review by iwpnd — CHANGES_REQUESTED** [review]
i checked out the PR, build and tested the viewer and did not find anything out of the ordinary. seems to be working just right.

some nitpicks. i would like to get rid of traces of mapbox in the naming and clean up comments that are still there left an right. but thats about it.

looks good 👍

**Review by ARolek — COMMENTED** [review]
Looking good in here! Thanks for tackling this. I don't have any additional comments beyond what @iwpnd pointed out.

**Review by ARolek — COMMENTED** [review]
Looking good in here! Thanks for tackling this. I don't have any additional comments beyond what @iwpnd pointed out.

File: ui/src/globals/store.js (hunk 1 above)
Comment (mapl) [reply ui/src/globals/store.js]: thanks for pointing out some obsolete mapbox references. There are still some more left, but these will be a topic for another commit.

**Comment by ARolek** [comment]
@iwpnd it looks like all your comments have been addressed. Would you mark the PR approved if you don't have any additional change requests?

**Review by iwpnd — APPROVED** [review]

**Comment by iwpnd** [comment]
my bad, wasn't aware that I was appointed a reviewer and was blocking this

**Pull request merged**